Examples/Examples.iml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@sohail-aspose Could you please clarify if _.iml_ files are needed? Also, this particular file references v19.4 of the SDK. Would you mind to add them to _.gitignore_ as well? Thanks.

Reply:
Done.